### ISSUE TITLE
 Duplicate booking & double charge

### ISSUE THREAD

**Issue by chris206** [206Tours/DB#1681]
![image](https://cloud.githubusercontent.com/assets/22382144/20760331/bb9c816c-b6ed-11e6-91f3-37d0933373ee.png)

Please fix whatever problem caused this and let me know if I will need to request refund on the broken one

**Comment by bradatkins** [comment]
She double clicked the complete order button...

**Comment by chris206** [comment]
Why is this not an automatic block? The idea that our database allows the same account to create more than one order in a 20 second period of time is crazy.

If this can be controlled for, is there any conceivable benefit to allowing that to happen?

**Comment by bradatkins** [comment]
Yes, we had some stuff in there, but it was generating problems with the
submission of orders. We haven't yet found a good stop-gap that is 100%.
The feel was we'd rather sometimes receive a duplicate order then no order
at all. That's not the end state, but it's currently that way.

**Comment by chris206** [comment]
If stopping the transaction creates problems, is there a way that we could at least break the code being sent for payment so that a charge doesn't go through? For example, this error cost us an unexpected $20.00 if the client decides that she doesn't want to be charged twice for deposit and insurance.

Even if the order must go through, does the charge if we can flag it?
